### ISSUE TITLE
Feature request: shared links dont work with wget, aria2 and curl

### ISSUE THREAD

**Issue by Kyuunex** [localsend/localsend#1704]
### Is your feature request related to a problem? Please describe.

I wanted to use this app to transfer a file to a home server. I did the obvious in the server shell:
```
wget http://192.168.1.103:53317
```
Which downloaded an HTML page and not the file

### Describe the solution you'd like

Something that works with `wget`, `aria2`, `curl`, etc. Maybe a toggle to enable direct linking.

### Describe alternatives you've considered

Alternative is spinning up a HTTP server manually, stitching together a link and doing `wget` that way.... which takes longer.

**Comment by Tienisto** [comment]
Yeah, this is the URL to get the HTML page.

Can you try `POST /api/localsend/v2/prepare-download`?
For more information: https://github.com/localsend/protocol?tab=readme-ov-file#52-receive-request-metadata-only

**Comment by Kyuunex** [comment]
I tried and it gave me a json you linked. 

Just typing all this out in the terminal, remembering the api endpoint from memory (next time i want to use this), copying the session IDs, is slower than spinning up an http server manually and typing the ip and filename.

I'm asking if we could get a toggle in settings that makes it when `wget http://192.168.1.103:53317` is invoked, download of the file immediately starts. There is **already** a toggle "Automatically accept requests", don't see a reason why when it's on, this does not work, and instead requires a browser with JavaScript enabled.

**Comment by Tienisto** [comment]
Because "Automatically accept requests" is meant to automatically accept the request on the sending device. The receiving device doesn't accept, it needs to **select** what files to download. I understand that it sounds silly if you only have 1 file though.

While it would be an improvement in your case (single file), what will happen if there are multiple files? The API is currently kept simple so that it works in every use case and not to be manually used by a human.

I would advice you to wait for the CLI in #11.
Here, it will be possible to download all files in parallel without zip / unzip (which is possible thanks to the current API design).

Adding the edge case (N=1) would overload the `/api/localsend/v2/prepare-download` route so it will complicate the implementation.

**Comment by Kyuunex** [comment]
Alright. Understood. 

The CLI version is out of scope for my usecase. I'll write my own something for this. 

Thanks.
